.goreleaser.yml

```diff
@@ -66,13 +66,9 @@ milestones:
 snapshot:
   name_template: "edge"
 
-before:
-  hooks:
-    - make crds-release-file
-
 release:
   extra_files:
-    - glob: ./build/out/crds.yaml
+    - glob: ./deploy/crds.yaml
```

Comment:
do we still want to upload it to the release?

Reply:
All of these files could be used directly from a GitHub tag...

Reply:
Doesn't seem necessary anymore. @sjberman what do you think?

Reply:
Probably not